eva-accession-clustering/src/main/java/uk/ac/ebi/eva/accession/clustering/configuration/batch/jobs/qc/ClusteringQCJobConfiguration.java

```diff
@@ -0,0 +1,139 @@
+/*
```

Comment:
As discussed in today's stand-up, I am not fully sure of making this a standard step for all species since we can't afford the querying/writing involved in the QC job. It is preferable to run this for the smaller species though.